C-Plus-Plus/dp/Maximum_area_Square_with_all_1.cpp

```diff
@@ -0,0 +1,59 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+int main()
+ {
+    int t;
+    cout<<"Enter the testcase \n";
+    cin>>t;
```

Comment:
Format the Code properly. Try to have spaces around the operators for better readability.